src/System.Private.ServiceModel/tests/Common/Scenarios/Endpoints.cs

```diff
@@ -145,6 +145,14 @@ public static string Tcp_VerifyDNS_Address
         }
     }
 
+    public static string Tcp_VerifyDNS_HostName
+    {
+        get
+        {
+            return BridgeClient.GetResourceHostName("WcfService.TestResources.TcpVerifyDNSResource");
+        }
+    }
+
```

Comment:
This seems like a weird of the Endpoints.cs class. Not blocking though, let's just get this in and we can talk about refactoring the BridgeClient classes